src/server/handlers/webhook.js

```diff
@@ -83,5 +115,23 @@ export const notify = async (req, res) => {
       logger.error(`Failed to request builds of ${repositoryUrl}: ${error}.`);
       return res.status(500).send();
     }
+  } else if (req.headers['x-launchpad-event-type'] === 'snap:build:0.1') {
```

Comment:
Refactored, thanks.

As for `snap:build:0.1`, yes, it's a versioning thing: see [LP's webhooks documentation](https://help.launchpad.net/API/Webhooks).  If that part of the event name changes then by definition it's a backward-incompatible change and we'll need to make some changes on our side, but since we subscribe specifically to `snap:build:0.1` we shouldn't be caught by surprise.